doc/bookmarks.txt

```diff
@@ -262,6 +262,10 @@ Use the location list to show all bookmarks (default 0):
 You can disable Ctrlp by setting (default 0)
 >
   let g:bookmark_disable_ctrlp = 1
+
+You can hidde line number and filename info in ctrlp window (default 1)
```

Comment:
Typo `hide`, but let's not have this option in the first place as previously stated.